tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/IntegrationTests/ConfigUnitTests.cs

```diff
@@ -76,7 +76,7 @@ public void GetAllUIConfigs()
 
             var logTableMoq = new Moq.Mock<IOptions<LogTables>>();
 
-            ConfigRepository configRepository = new ConfigRepository(generalSettingsMoq.Object, uiConfigsMoq.Object);
+            ConfigRepository configRepository = new ConfigRepository(null, generalSettingsMoq.Object, uiConfigsMoq.Object);

```

Comment:
Need to update the test code. Need to pass SPO authorization object and for time being setting to null